pkg/provenance.go

```diff
@@ -129,6 +132,13 @@ func verifyRootHash(ctx context.Context, rekorClient *client.Rekor, proof *model
 	if err := sth.UnmarshalText([]byte(*logInfo.SignedTreeHead)); err != nil {
 		return err
 	}
+	for _, inactiveShard := range logInfo.InactiveShards {
+		if *inactiveShard.TreeID == treeIDString {
+			if err := sth.UnmarshalText([]byte(*inactiveShard.SignedTreeHead)); err != nil {
+				return err
+			}
+		}
+	}
```

Comment:
nit: I don't think we can fix this but this seems odd that we would have to build the signed checkpoint ourselves; including iterating over inactive shards. It feels like this should be a method on the `logInfo`.

Reply:
Big +1, I think it should be part of params. I'll open a follow-up issue

Reply:
https://github.com/sigstore/rekor/issues/1091